src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -138,7 +138,7 @@ private static String getBusbarSectionId(Terminal terminal) {
         return connectedBusbarSectionFinder.getFirstTraversedBbsId();
     }
 
-    private static LineMapData toMapData(Line line) {
+    private static LineMapData toMapData(Line line, boolean withBusOrBusbarSection) {
```

Comment:
Maybe it's better an another method without this argument and with the default value `false` ?

Reply:
Ok, done